<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateEditBlock</class>
 <widget class="QMainWindow" name="CreateEditBlock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="blockNameLabel">
        <property name="text">
         <string>Block x</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Current use count : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="blockUseLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>334</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Occupancy Sensor:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sensorNameField">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a sensor number, System name or user name of a sensor. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Occupied sense:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>334</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="sensorDebounceGlobalCheck">
      <property name="text">
       <string>SensorDebounce</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Inactive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sensorDebounceInactiveField">
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Active</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sensorDebounceActiveField">
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="11" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Track color:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="trackColorBox">
        <item>
         <property name="text">
          <string>Black</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dark Gray</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gray</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Light Gray</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>White</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pink</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Orange</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="12" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Occupied Track color:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="occupiedColorBox">
        <item>
         <property name="text">
          <string>Black</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dark Gray</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gray</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Light Gray</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>White</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pink</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Orange</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="13" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Alternate track color:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="extraColorBox">
        <item>
         <property name="text">
          <string>Black</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dark Gray</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gray</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Light Gray</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>White</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pink</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Orange</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="14" column="0" colspan="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>334</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="15" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Memory Variable Name: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="memoryNameField"/>
      </item>
     </layout>
    </item>
    <item row="16" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Block Metric</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="metricField">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;set the cost for going over this block&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="17" column="0" colspan="2">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>334</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="18" column="0" colspan="2">
     <widget class="QCheckBox" name="permissiveCheck">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;quot;Is another train allowed to enter the block when it is already occupied&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Permissive Working Allowed</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0" colspan="2">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>334</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="20" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="blockEditDone">
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="blockEditCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="senseBox">
      <item>
       <property name="text">
        <string>Active</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inactive</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>357</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
